src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseDataFileService.cs

```diff
@@ -278,7 +287,12 @@ public async Task<Either<ActionResult, List<DataSetUploadResultViewModel>>> Uplo
                         .OnSuccess(async replacingFile
                             => await ValidateDataSetZip(releaseVersionId, zipFormFile, dataSetTitle, replacingFile))
                         .OnSuccess(async dataSet
-                            => await dataSetFileStorage.UploadDataSetsToTemporaryStorage(releaseVersionId, [dataSet], cancellationToken));
+                            => await dataSetFileStorage.UploadDataSetsToTemporaryStorage(releaseVersionId, [dataSet], cancellationToken))
```

Comment:
I notice duplication with a method above - are these due to be amalgamated into one process at some point in the future?